configure.ac

```
@@ -512,7 +540,8 @@ if test "x${lib_only}" = "xno" &&
 fi
 
 AM_CONDITIONAL([ENABLE_EXAMPLES], [ test "x${enable_examples}" = "xyes" ])
-AM_CONDITIONAL([ENABLE_EXAMPLE_QUICTLS], [ test "x${enable_examples}" = "xyes" && test "x${have_quictls}" = "xyes" ])
+AM_CONDITIONAL([ENABLE_EXAMPLE_QUICTLS], [ test "x${enable_examples}" = "xyes" && test "x${have_quictls}" = "xyes" -o "x${have_libressl}" = "xyes" ])
```

I think `-o "x${have_libressl}" = "xyes"` is not required because libressl has its own ENABLE_EXAMPLE_LIBRESSL below.

---

Just adding `|| ENABLE_EXAMPLE_LIBRESSL` to `if ENABLE_EXAMPLE_QUICTLS` in examples/Makefile.am is enough, I guess?

---

Thanks.  Still you should do at least something like so:

```suggestion
AM_CONDITIONAL([ENABLE_EXAMPLE_QUICTLS], [ test "x${enable_examples}" = "xyes" && (test "x${have_quictls}" = "xyes" || test "x${have_libressl}" = "xyes") ])
```